README.md

```diff
@@ -2,6 +2,21 @@
 
 AEGeAn: <b>a</b>nalysis and <b>e</b>valuation of <b>ge</b>nome <b>an</b>notations
 
+The AEGeAn Toolkit provides a bundle of tools for managing and analyzing whole-genome gene structure annotations.
+
+## Installation
+Please find detailed installation instructions and options in the
+[INSTALL](./INSTALL.md) document.
+
+
+## FAQ
+
+Please see
+[our wiki FAQ pages](https://github.com/BrendelGroup/AEGeAn/wiki/FAQ)
+for usage examples and suggestions.
```

Comment:
Same concern as before: I have no problem using the wiki if that's the way we want to go, but we should avoid having documentation split across different places. The `docs/` directory already has several pages explaining what programs are available, how to run them, GFF3 conventions, and so on.

Reply:
By the way, I forgot that I had ever put any content in the wiki. The notes on automated testing are a distant memory: probably a checklist I had planned on using to keep track of features needing testing. In any case, I never ended up using it and I'll be happy to delete it after six years of neglect.

Reply:
The idea behind the reorganization is to simplify matters and make maintenance easy. The readthedocs site is rather elaborate, and I'd be happy for you to maintain such document. But for someone wanting to install the code and run it, it's much better to have everything a click away from the home page, as is common with many github projects. The FAQ is just another place, easily accessible, where odd cases that come up can be documented.
Thus, I would suggest to simplify documentation, license, installation as per usual group style, but then for you to maintain a User's Manual as the readthedocs document, which can provide more detailed explanations on all the topics not covered in publications. Such an approach is also quite common, providing both the quick start right at hand and more details elsewhere.

Reply:
If we were to move the content of your new `INSTALL.md` file to the existing `docs/install.rst`, we could add a link to the rendered page to the README. That would certainly satisfy the "one click away" convenience, and the maintenance burden shouldn't be any different than for a markdown file in the root directory.

But taking a look at other recently updated git repos in the `BrendelGroup` space, I see what you mean by consistent group style. I'll treat the ReadTheDocs site as a user manual, and update the corresponding install instructions to point to the new `INSTALL.md` file as the endorsed installation procedure.